packages/storybook/package.json

```diff
@@ -53,7 +53,8 @@
     "@hig/theme-context": "^2.0.0",
     "@hig/theme-data": "^1.6.0",
     "faker": "^4.1.0",
-    "prop-types": "^15.6.1"
+    "prop-types": "^15.6.1",
+    "wait-port": "^0.2.2"
```

Comment:
might not want to add this into this PR if we don't know for sure that it has helped w/ the sauce labs/gemini testing. we should probably break this out into its own story.